src/UKNewsletterEpic/failure.stories.tsx

```diff
@@ -65,15 +65,11 @@ const StoryTemplate = (
                 subscribeToNewsletter={(newsletterId) => {
                     console.log(`subscribeToNewsletter invoked with id ${newsletterId}`);
                     return new Promise((resolve, reject) =>
-                        setTimeout(() => (args.simulateFailure ? reject() : resolve()), 1000),
+                        setTimeout(() => (args.simulateFailure ? reject() : resolve(' ')), 1000),
```

Comment:
Do we need to resolve with `' '` here?

Reply:
TS insists we have to include an argument in the resolve invocation. As it's Storybook I don't think it matters, as long as something is there?